test/functional/Java21Only/src/org/openj9/test/jep442/upcall/UpcallMHWithUnionStructTests.java

```diff
@@ -21,26 +21,30 @@
  */
 package org.openj9.test.jep442.upcall;
 
-import org.testng.annotations.Test;
 import org.testng.Assert;
-import org.testng.AssertJUnit;
-
-import java.lang.invoke.MethodHandle;
-import java.lang.invoke.VarHandle;
+import org.testng.annotations.Test;
 
 import java.lang.foreign.Arena;
-import java.lang.foreign.Linker;
 import java.lang.foreign.FunctionDescriptor;
 import java.lang.foreign.GroupLayout;
+import java.lang.foreign.Linker;
 import java.lang.foreign.MemoryLayout;
 import java.lang.foreign.MemoryLayout.PathElement;
```

Comment:
The locals on lines 987-988 are unused; please remove them and then the imports for `MemoryLayout.PathElement` and `VarHandle`.

Reply:
Fixed everywhere.